create_graphics.py

```diff
@@ -506,11 +506,17 @@ def graphics_driver(cla):
 
     # Create an empty zip file
     if cla.zip_dir:
-        zipfiles = {}
-        for tile in cla.tiles:
-            tile_zip_dir = os.path.join(cla.zip_dir, tile)
-            os.makedirs(tile_zip_dir, exist_ok=True)
-            zipfiles[tile] = os.path.join(tile_zip_dir, 'files.zip')
+        if cla.graphic_type == 'skewts':
+            zipf = None
+            os.makedirs(cla.zip_dir, exist_ok=True)
```

Comment:
These two lines (510 and 511) are unnecessary.

Reply:
I think one solution could be to consider a "skewt" a tile. Then you could use the previous chunk of code like this:

```
    if cla.zip_dir:
        zipfiles = {}
        tiles = cla.tiles if cla.graphic_type == "maps" else ['skewt']
        for tile in tiles:
            tile_zip_dir = os.path.join(cla.zip_dir, tile)
            os.makedirs(tile_zip_dir, exist_ok=True)
            zipfiles[tile] = os.path.join(tile_zip_dir, 'files.zip')
```

So that means you'd end up putting your SkewT zip files in `&DATAROOT;/@Y@m@d@H/nclprd/skewt/files.zip` if I am following the RTMA_3D.xml file correctly.